lib/localhost/authority.rb

```diff
@@ -176,6 +177,7 @@ def client_context(*args)
 		end
 		
 		def load(path = @root)
+			dir_migrate(path)
```

Comment:
Can we make the method name more descriptive, e.g.

```suggestion
			migrate_legacy_certificate_path(path)
```

Reply:
Sure! Went with `migrate_legacy_authority_path`